Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Integrations & Incidents Health Check",
     "description": "Do you know which of your integrations or open incidents failed? With this content, you can view your failed integrations and open incidents",
     "support": "xsoar",
-    "currentVersion": "1.1.15",
+    "currentVersion": "1.2.0",
```

Comment:
bugfix should be a revision usually

Reply:
I ran again with revision, also bumped to 1.2.0

Reply:
I ran again with revision, also bumped to 1.2.0